src/commons/yaml_parser.py

```diff
@@ -51,28 +52,28 @@ def convert_to_bytes(size):
     can be provided as byte(s), kb, kib, mb, mib, gb, gib, tb, tib
     :return equivalent bytes value for object size.
     """
-    kb = 1000
-    kib = 1024
```

Comment:
Instead of removing those, we can move this to constants file, since these values are used in corio.py and also in unit tests.

Reply:
done